code/src/components/app-snackbar/AppSnackbar.scss

```diff
@@ -12,4 +17,9 @@
   &__alert.MuiAlert-colorError {
     background-color: var(--spa-snackbar-error-bg-color, $red);
   }
+
+  & .MuiAlert-action,
+  & .MuiAlert-icon {
+    padding-block: 0;
```

Comment:
just `padding` not works here ? What is the differences ?

Reply:
> I investigated this style and find out that we can use better approach. I just modified line-height of `AppTypography` and it was enough. Do you like it?
> 
> ![image](https://private-user-images.githubusercontent.com/109922101/345037333-3890ce63-460a-47d0-affc-4501930eb9f8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk5MjU3OTgsIm5iZiI6MTcxOTkyNTQ5OCwicGF0aCI6Ii8xMDk5MjIxMDEvMzQ1MDM3MzMzLTM4OTBjZTYzLTQ2MGEtNDdkMC1hZmZjLTQ1MDE5MzBlYjlmOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzAyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwMlQxMzA0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZThiNTI2M2JlY2UzZGJlNDI0NDkwZmEyODg1ZDI5YjIxZDk1ZTI4NzhkNDhlMjdlNWQ4OGNhN2VhMzZmZDA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PKZzbiIxbBCV2gcWOXlyRBwsI4pBy0xN9tYzKEV18us)

Sure. Also make sense and looks nice. I just would like to know what is differences between this properties

Reply:
`padding-block: 0` is like `padding-y: 0`, adds padding to top and bottom and `padding: 0` adds `padding` to all sides